src/Symfony/Component/Mime/Part/DataPart.php

```diff
@@ -56,6 +56,10 @@ public static function fromPath(string $path, string $name = null, string $conte
             $contentType = self::$mimeTypes->getMimeTypes($ext)[0] ?? 'application/octet-stream';
         }
 
+        if (false === is_file($path)) {
```

Comment:
What do you think to put "is_readable"? And adapt the exception message

Reply:
@maxhelias done :)